### ISSUE TITLE
React (multiple): Some Lesson Overview items breaking TOP009 linting rule

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#29039]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Our [TOP009 custom linting rule](https://github.com/TheOdinProject/curriculum/blob/main/markdownlint/docs/TOP009.md) requires Lesson Overview items to be statements and not questions. They must also end with a period.

Some of our lessons in the React course have Lesson Overview items that are either missing a period at the end, or are written as questions. These will need to have punctuation fixed, and any questions rephrased as statements.

## Acceptance criteria

- [ ] The following lessons have their Lesson Overview items fixed according to our TOP009 linting rule:
  - [Introduction To React](https://www.theodinproject.com/lessons/node-path-react-new-introduction-to-react)
  - [React Components](https://www.theodinproject.com/lessons/node-path-react-new-react-components)
  - [Keys In React](https://www.theodinproject.com/lessons/node-path-react-new-keys-in-react)
  - [How To Deal With Side Effects](https://www.theodinproject.com/lessons/node-path-react-new-how-to-deal-with-side-effects)
  - [Component Lifecycle Methods](https://www.theodinproject.com/lessons/node-path-react-new-component-lifecycle-methods)
  - [React Router](https://www.theodinproject.com/lessons/node-path-react-new-react-router)
  - [Styling React Applications](https://www.theodinproject.com/lessons/node-path-react-new-styling-react-applications)

Please comment below if you'd like to be assigned to this issue. Please do not open a PR until you have been assigned by a maintainer.

### Path

Ruby / Rails, Node / JS

### Lesson Url

As above

**Comment by devnchill** [comment]
Can i give it a try ? I just have to make sure all the overview points start with a Capital Letter and end with "." instead of question mark right ?

**Comment by MaoShizhong** [comment]
@devnchill Any items that are questions will also need to be reworded as statements, since just swapping out the `?` for a `.` won't make grammatical sense.

**Comment by devnchill** [comment]
alright
